portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -548,6 +548,8 @@ definition {
 
 		Form.configureSubmitLabel(submitLabel = "!@#$%^&*()+?");
 
+		Button.click(button = "Done");
+
```

Comment:
@fmarcio This [commit](https://github.com/liferay-forms/liferay-portal/commit/ecf9cef5403341cea1e70150fa82683b280e7329) recently added to the `configureSubmitLabel` macro the `Button.clickDone();`.
Can you check if your change is still needed? If not, can you please remove it? Thanks!

Reply:
> @fmarcio This [commit](https://github.com/liferay-forms/liferay-portal/commit/ecf9cef5403341cea1e70150fa82683b280e7329) recently added to the macro `configureSubmitLabel` the `Button.clickDone();`.
> Can you check if your change is still needed? If not, can you please remove it? Thanks!

@carolmariaabb not needed anymore. Fixed! :-)